api/payment/index.js

```diff
@@ -12,4 +12,8 @@ router.post('', (req, res) => {
   stripe.charges.create(req.body, postCharge(res));
 });
 
+router.post('/connect', (req, res) => {
+  stripe.charges.create(req.body, postCharge(res));
+});
```

Comment:
Why have another route that's doing the same thing (as the one just above it)?

Reply:
That's a great question. I didn't think of it. This can be removed.